cli/src/commands/shared_recovery/info.rs

```diff
@@ -0,0 +1,63 @@
+use std::collections::HashMap;
+
+use itertools::Itertools;
+
+use crate::{build_main_with_client, utils::*};
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct Args {
+    }
+);
+
+build_main_with_client!(main, list_shared_recovery);
+
+pub async fn list_shared_recovery(_args: Args, client: &StartedClient) -> anyhow::Result<()> {
+    {
+        let mut spinner = start_spinner("Poll server for new certificates".into());
+        client.poll_server_for_new_certificates().await?;
+        spinner.stop_with_symbol(GREEN_CHECKMARK);
+    }
+
+    let info = client.get_self_shamir_recovery().await?;
+    let users = client.list_users(false, None, None).await?;
+    let users: HashMap<_, _> = users.iter().map(|info| (info.id, info)).collect();
+
+    match info {
+                libparsec_client::SelfShamirRecoveryInfo::Deleted {
+                    deleted_on,
+                    deleted_by,
+                    ..
+                } => println!("{RED}Deleted{RESET} shared recovery - deleted by {deleted_by} on {deleted_on}"),
+                libparsec_client::SelfShamirRecoveryInfo::SetupAllValid {
+                    per_recipient_shares, threshold, ..
+                } => println!("Shared recovery {GREEN}set up{RESET} with threshold {threshold}\n{}", per_recipient_shares.iter().map(|(recipient, share)| {
+                    // this means that a user disappeared completely, it should not happen
+                    let user= &users.get(recipient).expect("missing recipient").human_handle;
+                    format!("\t• User {user} has {share} share(s)", // TODO: special case if there is only on share
```

Comment:
```suggestion
                    format!("• User {user} has {share} share(s)", // TODO: special case if there is only on share
```

Other lists are not indented